### PR TITLE
front: fix speed multiplier input in op study

### LINKED ISSUE #5986: front: deleting the content of the "speed multiplier" value creates a 0 which cannot be deleted
### What happened?

Sorry the title is not very clear, here is the video:
https://github.com/osrd-project/osrd/assets/115472753/f2312336-3ec3-457d-a257-1c09df58c7c0

### What did you expect to happen?

no 0 should appear, or at least it should be deletable

### How can we reproduce it (as minimally and precisely as possible)?

1. go to study & create a train
2. delete the value in the "speed multiplier" field and try to delete the 0 that is created

### What operating system, browser and environment are you using?

- **Browser**: Chrome Version 119.0.6045.199
- **OS**: Windows 11
- **Env**: https://rec-osrd.reseau.sncf.fr/

### OSRD version (top right corner `Account` button > `Informations`)

staging 3df6fbc

### PR DESCRIPTION
**Pull request by RomainValls**
closes #5986

### CODE REVIEW

**Review by SharglutDev — CHANGES_REQUESTED** [review]
Thank you for your PR.

It seems that without the min in the input, we can do that with a copy/paste. As it wasn't possible before, maybe we should check with @nicolaswurtz if this behavior is ok.

Capture d’écran 2023-12-14 à 09 38 21

File: front/src/modules/simulationResult/components/TimeButtons.tsx [hunk 1]
```diff
@@ -118,9 +123,10 @@ const TimeButtons = ({ selectedTrain, timePosition }: TimeButtonsProps) => {
       <InputSNCF
         noMargin
         type="number"
+        min={1}
         id="simulation-speed"
-        value={simulationSpeed}
-        onChange={(e) => changeSimulationSpeed(Number(e.target.value))}
+        value={simulationSpeed !== null ? simulationSpeed : ''}
```
Comment (anisometropie) [inline front/src/modules/simulationResult/components/TimeButtons.tsx]:
```suggestion
        value={simulationSpeed ?? ''}
```

**Review by anisometropie — CHANGES_REQUESTED** [review]

**Review by Akctarus — COMMENTED** [review]
Need to fix previous comments before approving

**Comment by nicolaswurtz** [comment]
I don't understand what is precisely the question, but if I guess right, the new behavior will allow negative values. This shouldn't be allowed. Min should be 1.

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested. Just squash your commits please as they aren't anything in commit 2 and 3.

**Review by anisometropie — COMMENTED** [review]
This bug should not but handled manually at this level, this should be handle by a dedicated component that we use everytime we need to have an Input with numbers

**Review by nicolaswurtz — APPROVED** [review]

**Comment by nicolaswurtz** [comment]
> This bug should not but handled manually at this level, this should be handle by a dedicated component that we use everytime we need to have an Input with numbers

Yes! It will be part of future input component from `ui-core`.

**Review by anisometropie — APPROVED** [review]

**Pull request merged**